bin/setup

```diff
@@ -8,6 +8,14 @@ def run(command)
   abort "command failed (#{$?}): #{command}" unless system command
 end
 
+def copy_default_file(src, dest)
+  if File.exist?(dest)
+    puts "Skipping cp #{src.inspect} to #{dest.inspect}, target already exists"
+    return
+  end
+  run "cp '#{src}' '#{dest}'"
+end
```

Comment:
This doesn't seem to be related to this PR. Also, I'm not sure we want to do this. If someone changes `application.yml.example`, I want my local `application.yml` to be overwritten when running `make setup` to make sure I'm using the latest settings.

Reply:
I did this because I symlinked `application.yml.example` to `application.yml` since most people don't customize this file at all. And the setup script explodes if you have previously symlinked it, so I wasn't able to make forward progress without doing this.